Content.Client/UILinks.cs

```diff
@@ -1,11 +1,12 @@
-namespace Content.Client
+﻿namespace Content.Client
 {
     public static class UILinks
     {
         public const string GitHub = "https://github.com/space-wizards/space-station-14/";
         public const string Patreon = "https://www.patreon.com/spacestation14";
 
         public const string Discord = "https://discordapp.com/invite/t2jac3p";
+        public const string BugReport = "https://github.com/space-wizards/space-station-14/issues/new/choose";
```

Comment:
Should this go to the issues page instead? Just so they have a chance to search if it's a known issue rather than rolling their face on their keyboard.